OMFAPI/Program.cs

```diff
@@ -60,14 +60,23 @@ public static bool RunMain(bool test = false, Dictionary<string, dynamic> lastSe
                 {
                     if ((endpoint.VerifySSL is bool boolean) && boolean == false)
                     {
-                        Console.WriteLine("You are not verifying the certificate of the end point.  This is not advised for any system as there are security issues with doing this.");
+                        if (string.Equals(endpoint.EndpointType, "ADH", StringComparison.OrdinalIgnoreCase))
+                        {
+                            // The certificate check should not fail for AVEVA Data Hub since that certificate is managed by AVEVA. If the certificate verification is failing, please contact Technical Support.
+                            Console.WriteLine($"Certificate verification should not be diabled for AVEVA Data Hub endpoints. This setting will be ignore for namespace {endpoint.NamespaceId}.");
+                        }
+                        else if (string.Equals(endpoint.EndpointType, "PI", StringComparison.OrdinalIgnoreCase))
+                        {
+                            Console.WriteLine($"You are not verifying the certificate of the PI end point for Data Archive {endpoint.DataArchiveName}.  This is not advised for any system as there are security issues with doing this.");
 
-                        // Create custom callback to disable certificate checks, create an HttpClient object using it
-                        endpoint.Handler = new ();
-                        endpoint.Handler.ServerCertificateCustomValidationCallback = (sender, certificate, chain, sslPolicyErrors) => true;
-                        endpoint.Client = new (endpoint.Handler);
+                            // Create custom callback to disable certificate checks, create an HttpClient object using it
+                            endpoint.Handler = new ();
+                            endpoint.Handler.ServerCertificateCustomValidationCallback = (sender, certificate, chain, sslPolicyErrors) => true;
+                            endpoint.Client = new (endpoint.Handler);
+                        }  
```

Comment:
What about EDS?

Reply:
I wasn't sure what to do with that one because it's not https. so even if you add a callback that short circuits the verification, it shouldn't even be called in the first place because there's no certificate presented. But I should test it out to verify instead of just assuming. Either way, the property is not in the EDS section of the appsettings placeholder example, so people shouldn't be using it, but since all three share a common endpoint class it is theoretically possible for someone to set it anyway.